src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,124 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static RegistrationServiceImpl registrationService;
+    private StorageDaoImpl storageDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        storageDao = new StorageDaoImpl();
+    }
+
+    public User createUser(String login, String password, int age) {
+        User user = new User();
+        user.setLogin(login);
+        user.setPassword(password);
+        user.setAge(age);
+        return user;
+    }
+
+    @Test
+    void register_Check_Age_Ok() {
+        User user1 = createUser("john1993", "123456", 20);
+        User user2 = createUser("jack1993", "123456", 18);
```

Comment:
Let me recommend a better approach - in setUp method set up a default valid user (it'll be a field). Then use it everywhere, overriding necessary attributes (making password too short for a test for example)